Orange/widgets/classify/tests/test_owlogisticregression.py

```diff
@@ -14,7 +12,6 @@ def test_coef_table_single(self):
         learn = LogisticRegressionLearner()
         classifier = learn(data)
         coef_table = create_coef_table(classifier)
-        self.assertEqual(1, len(bn.stats(coef_table.metas, None)))
```

Comment:
Although I have no idea what this assert checked, I would like to know why it does not have to test it any more